content/glossary.md

```diff
@@ -462,6 +465,9 @@ Pertains to the rules and guarantees about how messages are delivered among diff
 * *Total Order Delivery*: This ensures a consistent global order in which messages are delivered. If one correct process delivers message $a$ before message $b$, then all correct processes will deliver $a$ before $b$. That is, there's a universally agreed order: either $a$ comes before $b$ or $b$ comes before $a$.
 * *Causal Order Delivery*: The order preserves the sequence of messages based on their cause-and-effect relationship. If a message "$a$" is delivered before message "$b$" is sent, then "$a$" will always precede "$b$". Likewise, if a sender sends message "$b$" before dispatching message "$c$", "$c$" will always come after "$b$".
 
+### Oracle
+An oracle is a component that facilitates communication between blockchain networks and external data sources. It provides smart contracts with access to real-world information, events, or conditions that are outside the native blockchain, enabling them to execute decisions based on external data. Oracles are crucial for enhancing the functionality and applicability of smart contracts across multiple real-world use cases.
```

Comment:
"them" is unclear; I suggest "…enabling the chain to…"

Reply:
Also a good suggestion.